src/ResourceManager/Common/Commands.Common.Authentication.ResourceManager/AzureRmProfile.cs

```diff
@@ -566,6 +566,7 @@ public bool TryRemoveEnvironment(string name, out IAzureEnvironment environment)
 
         public bool TryCopyProfile(AzureRmProfile other)
         {
+            this.DefaultContextKey = other.DefaultContextKey;
```

Comment:
I think this belongs after the contexts are copied from the remote profile

Reply:
Hey Mark. I can move it to after the contexts are copied. I think I tested this to see if it would make a difference and I don't think it did. Anyways, I'll push a commit in a bit to fix this..